.github/workflows/coverage.yml

```diff
@@ -49,22 +49,22 @@ jobs:
           echo "$COVERAGE_DETAILS_OUTPUT"
 
       - name: Find Comment
-        uses: peter-evans/find-comment@v2
+        uses: peter-evans/find-comment@v3
         id: find-comment
         continue-on-error: true
         with:
           issue-number: ${{ github.event.pull_request.number }}
           comment-author: 'github-actions[bot]'
-          body-includes: Coverage Results
+          body-includes: Coverage Summary
```

Comment:
"Results" was in the search string but not in the actual output. Changed both to "Coverage Summary"